nestkernel/connection_manager.cpp

```diff
@@ -964,6 +964,7 @@ nest::ConnectionManager::data_connect_connectome( const ArrayDatum& connectome )
     Node* source_node = kernel().node_manager.get_node( source_gid );
     connect_( *source_node, *target_node, source_gid, thr, syn_id, cd );
   }
+  kernel().connection_manager.set_have_connections_changed( true );
```

Comment:
@hakonsbm For consistency with the other cases, I would suggest to move this call to the beginning of the method. In that way the flag is set even if an exception should occur after the first connection has been created.